nerdtree_plugin/fs_menu.vim

```diff
@@ -203,7 +203,7 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        let metadata = split(system('ls -ldh ' . shellescape(treenode.path.str())), '\n')
```

Comment:
@olivier-o I actually like seeing the exact size of the file, but to make it easier to read, what do you think of this option?

```
let metadata = split(system("ls -ld " . shellescape(treenode.path.str()) . " | sed -e :a -e 's/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/;ta'"), '\n')
```

Props to [Brian Cooper](http://stackoverflow.com/users/695419/brian-cooper) on StackOverflow for this [answer](http://stackoverflow.com/a/23666338/510067).